_posts/2017/05/2017-05-15-EmpoweringOthers.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+authors: ["Rayna Harris"]
+title: "What is the reward for empowering others?"
+date: 2017-05-14
+time: "20:00:00"
+category: ["Community","Teaching","Education","Workshops","Mentoring"]
+---
+
+What is the reward for empowering others? Following a few round-table discussions with Software Carpentry instructors, I gained some insight into what drives us to enthusiastically volunteer our time. I came to the conclusion that many of us share a common sense of accomplishment when we empower others. We enjoy empowering others through mentoring and education. Software Carpentry gives us a fountain-full of opportunities for empowering others through community developed lessons, community supported workshops, and world-wide community conversations. We start out volunteering a little when there i time, but we soon carve out more and more space in our schedule for Software Carpentry events and activities that empower us to empower others.  
+
+I can't help but wonder if part of this is driven by a lack of reward for empowering others in the traditional academic setting. Compare the motivation of learners in a required semester-long class introductory class to learners in a intense two-day workshop teaching best practices for domain-specific research. I prefer to teach the later class because I'm certain it enhances my learner's education and research. Furthermore, we all know that many academic professors are rewarded for research while paid for teaching (something I know but still don't quite comprehend). It is awesome that the Software Carpentry Instructor is valued for  contributions in teaching that enable the research that furthers the careers of our learners. 
```

Comment:
1. "if part of this is driven" => "if this is driven in part"
2. "in traditional academic settings"
3. second sentence: OK, I'll compare them. And...?  (I think you should say explicitly that learners in a two-day workshop are more highly motivated...)
4. Third sentence: "learners'" not "learner's" since it's plural
5. Is the third sentence implying that regular classes *don't* enhance learners' education and research?
6. Just "professors", not "academic professors", since all professors are academic.
7. Final sentence might be stronger if you said, "For many of us, Software Carpentry is where we hear most clearly that our teaching really is valued."